File: Tables/TablesProgram.go
package main
import "fmt"
func main(){
   var n int
   fmt.Print("Enter the number to print the multiplication table:")
   fmt.Scanf("%d", &n)
   for i:=1; i<11; i++ {
      fmt.Println(n, "x", i, "=", n*i)
   }
}

File: Tables/TablesProgram.c
#include <stdio.h>
int main() {
    int n;
    printf("enter the number of tables: ");
    scanf("%d",&n);
    for(int i=1;i<=10;i++) {
    printf("%dx%d=%d \n",n,i,n*i);
    }
    return 0;
}